pkg/gardenlet/controller/shoot/shoot_control.go

```diff
@@ -370,7 +370,7 @@ func (c *Controller) isSeedAvailable(seed *gardencorev1beta1.Seed) error {
 		return fmt.Errorf("seed is set for deletion")
 	}
 
-	return health.CheckSeed(seed, c.identity)
+	return health.CheckSeedForMigration(seed, c.identity)
```

Comment:
Can you rename the `isSeedAvailable` function to `isSeedReadyForMigration` or something similar? It wasn't clear from the name and I had to check where this thing is actually called.